src/TodoItem.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function TodoItem({item, deleteTodo, changeCompleteTodo}){
+    const date = (item.date).toDateString();
+    const done = item.isDone;
+    function onDeleteTodo(){
+        deleteTodo(item.id)
+    }
+    function onChangeComplete(){
+        changeCompleteTodo(item);
+    }
+    return (
+        <li style={{'textDecoration' : done ? 'line-through' : 'none'}} className="todo-app__item">
```

Comment:
Это очень плохо читается. Почему ты ставишь там пробелы? :)

```suggestion
        <li style={{'textDecoration': done ? 'line-through' : 'none'}} className="todo-app__item">
```